agent/agent-tooling/src/main/java/io/opentelemetry/sdk/metrics/ViewBuilderAccessor.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.opentelemetry.sdk.metrics;
+
+import io.opentelemetry.sdk.metrics.internal.view.AttributesProcessor;
+
+public class ViewBuilderAccessor {
```

Comment:
should we move this to standalone exporter module so that it can be reused?

Reply:
I think it's too early to use in the standalone exporter module, because `AttributesProcessor` is in an internal package, and we risk causing version conflicts for users if we use internal classes (which isn't the case in the javaagent since we have control of the SDK version used)

Reply:
added to this week's SIG meeting to discuss 👍